lua/telekasten.lua

```diff
@@ -1629,12 +1629,12 @@ local function FindFriends(opts)
 
     vim.cmd("normal yi]")
     local title = vim.fn.getreg('"0')
-    title = title:gsub("^(%[)(.+)(%])$", "%2")
+    title = string.sub(title, 0, string.find(title, "%s*|") - 1)
```

Comment:
I do not think this works on non-alias links. In my tests this throws an error due to a nil value...